python-meh.spec

```diff
@@ -70,79 +44,60 @@ Requires: libreport-cli >= %{libreportver}
 Requires: libreport-python3 >= %{libreportver}
 Requires: python3-six
 
-%description -n python3-meh
+%description -n python3-%{srcname}
 The python3-meh package is a python 3 library for handling, saving, and reporting
 exceptions.
 
-%package -n python3-meh-gui
+%package -n python3-%{srcname}-gui
 Summary: Graphical user interface for the python3-meh library
-Requires: python3-meh = %{version}-%{release}
+Requires: python3-%{srcname} = %{version}-%{release}
 Requires: python3-gobject, gtk3
 Requires: libreport-gtk >= %{libreportver}
 
-%description -n python3-meh-gui
+%description -n python3-%{srcname}-gui
 The python3-meh-gui package provides a GUI for the python3-meh library.
 
-%endif
-
 %prep
 %setup -q
 
-%if 0%{?with_python3}
 rm -rf %{py3dir}
 cp -a . %{py3dir}
-%endif # with_python3
 
 %build
 make
 
-%if 0%{?with_python3}
 pushd %{py3dir}
 make PYTHON=%{__python3}
 popd
-%endif
 
 %check
 make test
 
-%if 0%{?with_python3}
 pushd %{py3dir}
 # Needs UTF-8 locale
 LANG=en_US.UTF-8 make PYTHON=%{__python3} test
 popd
-%endif
 
 %install
 rm -rf %{buildroot}
 make DESTDIR=%{buildroot} install
 
-%if 0%{?with_python3}
 pushd %{py3dir}
 make PYTHON=%{__python3} DESTDIR=%{buildroot} install
 popd
-%endif
 
 %find_lang %{name}
 
 %clean
 rm -rf %{buildroot}
 
-%files -f %{name}.lang
+%files -n python3-%{srcname} -f %{name}.lang
 %defattr(-,root,root,-)
 %doc ChangeLog COPYING
-%{python_sitelib}/*
-%exclude %{python_sitelib}/meh/ui/gui.py*
-
-%files gui
-%{python_sitelib}/meh/ui/gui.py*
-%{_datadir}/python-meh
-
-%files -n python3-meh
-%doc ChangeLog COPYING
 %{python3_sitelib}/*
 %exclude %{python3_sitelib}/meh/ui/gui.py*
 
-%files -n python3-meh-gui
+%files -n python3-%{srcname}-gui
 %{python3_sitelib}/meh/ui/gui.py*
```

Comment:
Add the corresponding `__pycache__` files as well. No need for the `*`.

Reply:
Changed as well.